### PR TITLE
Checkbox, Radio: animation according to guideline

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/849
[Problem] Animation is wrong
[Solution]
 - create sprites for checkbox and radio
 - prepare css animations

[Examples]
UIComponents > Controllers > CheckBox (List)
UIComponents > Controllers > CheckBox (Image Thumbnail)
UIComponents > Controllers > CheckBox (List Selected)

[Demo]
checkbox
radio

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: src/css/profile/mobile/common/checkbox.less [hunk 1]
```diff
@@ -31,15 +41,24 @@ input[type="checkbox"].ui-checkbox {
 		bottom: 0;
 		opacity: 0.8;
 		background-color: var(--control-inactive-color);
-		mask-image: url("images/3_Controllers/Checkbox/sem_checkedtextview_check_to_on_mtrl_000.svg");
-		mask-size: 100%;
-		mask-position: center;
+		animation-fill-mode: both;
```
Comment (korneliakobiela) [inline src/css/profile/mobile/common/checkbox.less]: Can we use shorthand for animation?

File: src/css/profile/mobile/common/checkbox.less [hunk 2]
```diff
@@ -31,15 +41,24 @@ input[type="checkbox"].ui-checkbox {
...
+		animation-name: none;
+		animation-duration: 250ms;
+		animation-timing-function: steps(26);
+		animation-direction: normal;
+		width: 100%;
+		height: 100%;
+		mask-image: url("images/3_Controllers/Checkbox/sprites/sem_checkedtextview_check_to_on_mtrl_sprites.svg");
+		mask-size: auto 100%;
+		mask-repeat: no-repeat;
+		mask-position: 0% 0;
```
Comment (korneliakobiela) [inline src/css/profile/mobile/common/checkbox.less]: Isn't it more elegant to make 
``` css
mask-position: 0 0;
```
?

File: src/css/profile/mobile/common/checkbox.less (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/checkbox.less]: I was wondering about it. I removed `name` property in this style. I'll left this with specific properties. I don't need completed animation definition in this place.

File: src/css/profile/mobile/common/checkbox.less (hunk 2 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/checkbox.less]: Done.

**Review by korneliakobiela — APPROVED** [review]
LGTM

**Pull request merged**